easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'splitRef'
+version = '0.0.2'
+
+homepage = 'http://www.sph.umich.edu/csg/yli/splitRef/index.html'
+description = """ splitRef splits a reference haplotype file into smaller files 
+ with subsets of markers. The current version is a pre-release. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.sph.umich.edu/csg/yli/splitRef/download/']
+sources = ['%s.V%s.tgz' % (name, version.replace(".",""))]
+
+sanity_check_paths = {
+    'files': ["splitRef_V002/splitRef.pl"],
+    'dirs': ["examples"]
+}
+
+# add folder splitRef_V002 to PATH
+modextrapaths = { 
+    'PATH': '%s_V%s' % (name, version.replace(".",""))
+}
+
+moduleclass = 'bio'
+
```

Comment:
remove last line, it has the nice benefit of triggering a recent jenkins test (if/when that works!)

lgtm , put this in REVIEW plz!